CHANGELOG.md

```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [1.2.0] - 2024-07-01
 ### Changed
 - Update Astarte logo in the login page and in the sidebar.
+- Navigate to requested path after a successful login.
```

Comment:
This release was already released. We can't add things in the past.
Usually the first PR of the new dev cycle also initializes the neccessary files. You can pretty much copy-paste this commit and use `[1.2.1] - Unreleased` in the changelog and `1.2.1-dev` in package files
https://github.com/astarte-platform/astarte-dashboard/commit/0aaf59f925d377dd116e9723bc396c5edd51453f